docs/module_web_with_python.md

```diff
@@ -5,137 +5,437 @@
 ### Clean code
 
 #### Point out 5 suggestions, how to format an SQL query!
+* Separate your query into multiple lines.
+* Capitalize keywords such as "WHERE, HAVING, SELECT" for better readability.
+* Use "AS" to rename columns when needed for a prettier display.
+* Use underscore when naming tables/columns with multiple words in them.
+* Avoid giving the same name for both the table and a column.
 #### What layers can you name in a simple web application?
```

Comment:
Might worth to note other form of layers:
Presentation,
Data service,
Business,
Data access

Reply:
fixed